baseplate/metrics.py

```diff
@@ -261,6 +272,28 @@ def decrement(self, delta=1, sample_rate=1.0):
         """
         self.increment(delta=-delta, sample_rate=sample_rate)
 
+class Histogram(object):
+    """A bucketed distribution of integer values across a specific range.
+
+    Records data value counts across a configurable integer value range
+    with configurable buckets of value precision within that range.
+
+    Configuration of each histogram is managed by the backend service,
+    not by this interface. This implementation depends on histograms
+    being created for timer values in the supporting StatsD backend.
+    """
+    def __init__(self, transport, name):
+        self.transport = transport
+        self.name = name
+
+    def add_sample(self, value):
+        """Add a new value to the histogram.
+
+        This records a new value to the histogram; the bucket it goes in
+        is determined by the backend service configurations.
+        """
+        serialized = self.name + (":{:h}|ms".format(value).encode())
```

Comment:
Er, I thought the `ms` would turn into `h` not the python side stuff?

Reply:
Ah whoops!